pandas/core/arrays/sparse/array.py

```diff
@@ -1811,6 +1811,21 @@ def _formatter(self, boxed: bool = False):
         # This will infer the correct formatter from the dtype of the values.
         return None
 
+    # ------------------------------------------------------------------------
+    # GroupBy Methods
+
+    def _groupby_op(
+        self,
+        *,
+        how: str,
+        has_dropped_na: bool,
+        min_count: int,
+        ngroups: int,
+        ids: npt.NDArray[np.intp],
+        **kwargs,
+    ):
+        raise NotImplementedError(f"{self.dtype} dtype not supported")
```

Comment:
Shouldn't the base class raise a `NotImplementedError`? Or is this to keep the same error message?

Reply:
This is just to keep the existing message

Reply:
Looks like this isn't actually needed.  Will remove in a follow-up (or next commit if it turns out to be necessary)